### DISCUSSION TITLE
G Helper no longer starting on bootup?

### DISCUSSION THREAD

**Discussion by Draebor** [seerge/g-helper#742] [Q&A] [answered]
running latest 0.95.0.0 version.

app doesnt want to start on bootup automatically despite run on startup being checked?

i have to manually open it to get it to run.

**Comment by ke1ne** [comment, marked as answer]
Try to run it as Administrator, re-add it to autorun via the checkbox and restart your laptop.

**Reply by Khalahari** [reply]
how to " re-add it to autorun via the checkbox " ?

**Reply by seerge** [reply]
Uncheck and check startup checkbox:)

**Reply by Khalahari** [reply]
Sure, just did, thanks for the reply. I thought it was inside of windows settings. I`ll check if work-out later.

**Comment by Draebor** [comment]
cheers, sorted.  thanks!
